packages/core/src/codewhisperer/service/recommendationService.ts

```diff
@@ -22,10 +29,33 @@ export class RecommendationService {
         return this._isRunning
     }
 
+    private _onSuggestionActionEvent = new vscode.EventEmitter<SuggestionActionEvent>()
+    get suggestionActionEvent(): vscode.Event<SuggestionActionEvent> {
+        return this._onSuggestionActionEvent.event
+    }
+
+    private _acceptedSuggestionCount: number = 0
+    get acceptedSuggestionCount() {
+        return this._acceptedSuggestionCount
+    }
+
+    private _totalValidTriggerCount: number = 0
+    get totalValidTriggerCount() {
+        return this._totalValidTriggerCount
+    }
+
```

Comment:
these seem unused, can we remove them?

Reply:
hey Max, they will be used later, example is here https://github.com/aws/aws-toolkit-vscode/pull/4335/files#diff-924d011d048deada898a06ebee916f9687f4243f3ce6585dcdb147bf0d27ad9dR70